pymatviz/ptable.py

```diff
@@ -703,293 +751,208 @@ def histogram(
-    show_scale: bool = True,
-    show_values: bool = True,
-    infty_color: str = "lightskyblue",
-    na_color: str = "white",
-    heat_mode: Literal["value", "fraction", "percent"] | None = "value",
-    fmt: str | Callable[..., str] | None = None,
-    cbar_fmt: str | Callable[..., str] | None = None,
-    text_color: str | tuple[str, str] = "auto",
-    exclude_elements: Sequence[str] = (),
-    zero_color: str = "#eff",  # light gray
-    zero_symbol: str | float = "-",
-    text_style: dict[str, Any] | None = None,
-    label_font_size: int = 16,
-    value_font_size: int = 12,
-    tile_size: float | tuple[float, float] = 0.9,
-    f_block_voffset: float = 0.5,
-    hide_f_block: bool | None = None,
-    **kwargs: Any,
 ) -> plt.Axes:
-    """Plot a heatmap across the periodic table of elements.
+    """Plot a heatmap across the periodic table.
 
     Args:
-        values (dict[str, int | float] | pd.Series | list[str]): Map from element
-            symbols to heatmap values or iterable of composition strings/objects.
-        log (bool | Normalize, optional): Whether colormap scale is log or linear. Can
-            also take any matplotlib.colors.Normalize subclass such as SymLogNorm as
-            custom color scale. Defaults to False.
-        ax (Axes, optional): matplotlib Axes on which to plot. Defaults to None.
-        count_mode ('composition' | 'fractional_composition' | 'reduced_composition'):
-            Reduce or normalize compositions before counting. See count_elements() for
-            details. Only used when values is list of composition strings/objects.
-        cbar_title (str, optional): Color bar title. Defaults to "Element Count".
-        cbar_range (tuple[float | None, float | None], optional): Color bar range.
-            Can be used e.g. to create multiple plots with identical color bars for
-            visual comparison. Defaults to automatic based on data range.
-        cbar_coords (tuple[float, float, float, float], optional): Color bar position
-            and size: [x, y, width, height] anchored at lower left corner. Defaults to
-            (0.18, 0.8, 0.42, 0.05).
-        cbar_kwargs (dict[str, Any], optional): Additional keyword arguments passed to
-            fig.colorbar(). Defaults to None.
-        colorscale (str, optional): Matplotlib colormap name to use. Defaults to
-            "viridis". See https://matplotlib.org/stable/users/explain/colors/colormaps
-            for available options.
-        show_scale (bool, optional): Whether to show the color bar. Defaults to True.
-        show_values (bool, optional): Whether to show the heatmap values in each tile.
-            Defaults to True.
-        infty_color: Color to use for elements with value infinity. Defaults to
-            "lightskyblue".
-        na_color: Color to use for elements with value infinity. Defaults to "white".
-        heat_mode ("value" | "fraction" | "percent" | None): Whether to display heat
```

Comment:
@janosh Can I have a quick explaination on this argument please?

https://github.com/janosh/pymatviz/blob/d222982c423a78e69914f3ae67942bc44ecf10ba/pymatviz/ptable.py#L765-L769

What is the difference between "fraction" mode and "percent"?

Any what does `None` do? Does `None` hide the values altogether, but we already have `show_values` for this?

Reply:
`percent` and `fraction` only differ by a factor of 100 and a suffix of `%` (IIRC). so `fraction` would render as `0.025` and `percent` as `2.5%`. and yes, `None` should hide the values altogether. it's possible that that's redundant with `show_values` now. i may have forgotten about the `heat_mode=None` option, when i added `show_values` 😅

Reply:
BTW, what do you think of the idea to rename `heat_mode` (which is a bit ambiguous to me personally) to `values_mode` (for "values display mode")? Now that we are grouping arguments.
```python
# Values
show_values: bool = True,
heat_mode: Literal["value", "fraction", "percent"] = "value",  # values_mode maybe?
values_pos: tuple[float, float] | None = None,
values_color: str = "black",  
values_kwargs: dict[str, Any] | None = None,
```

Reply:
i'm definitely open to a change. we can make it backwards compatible with a deprecation warning by still looking for `heat_mode` in `kwargs` and renaming it to `values_display_mode` if found.

though come to think of it, maybe `show_value_mode` is similarly clear and a little shorter than `values_display_mode`. `show_value_mode` could then entirely replace `show_values`. the previous `show_values=True|False` would become `show_value_mode='value'|'off'`. so in total, we'd have `show_value_mode: Literal["value", "fraction", "percent", "off"]`

Reply:
Yes! Let's combine these two args into a single one :)